syntax/gomod.vim

```diff
@@ -61,7 +61,7 @@ syntax match gomodToolchainVersion "go1\(.\d\+\)\{,2\}\(rc\d\+\)\?\([ \t-].*\)\?
 highlight default link gomodToolchainVersion Identifier
 
 " match go versions
-syntax match gomodGoVersion "1\.\d\+" contained
+syntax match gomodGoVersion "\(1.\d\+\)\(\(.\d\+\)\|\(rc\d\+\)\)\?" contained
```

Comment:
I think this is clearer with magic: `\v(1.\d+)((.\d+)|(rc\d+))?`. I think I've seen this used elsewhere in these syntax files so maybe a worthwhile change?

Reply:
I'm ok without the magic.